lib/elb/models/AccessLogDiff.rb

```diff
@@ -0,0 +1,55 @@
+require "common/models/Diff"
+require "util/Colors"
+
+module Cumulus
+  module ELB
+    # Public: The types of changes that can be made to an access log config
+    module AccessLogChange
+      include Common::DiffChange
+
+      ENABLED = Common::DiffChange.next_change_id
+      BUCKET = Common::DiffChange.next_change_id
+      EMIT = Common::DiffChange.next_change_id
+      PREFIX = Common::DiffChange.next_change_id
+    end
+
+    # Public: Represents a single difference between local configuration and
+    # an AWS Load Balancer Access Log.
+    class AccessLogDiff < Common::Diff
+      include AccessLogChange
+
+      def asset_type
+        "Access Log Config"
+      end
+
+      def diff_string
```

Comment:
So, because all these are the same, you could do

```ruby
first_line = case @type
when ENABLED
  "Enabled:"
when BUCKET
  "S3 Bucket:"
...

[
  first_line,
  ...
].join("\n")
```